R/config.R

```diff
@@ -1,10 +1,23 @@
 
 
 rsconnectConfigDir <- function(subDir = NULL) {
-  applicationConfigDir("connect", subDir)
+
+  # first check whether the main rsconnect directory exists
```

Comment:
Is there possible confusion about `rsconnectRootPath` versus `rsconnectConfigDir`? Both create directories named "rsconnect", but in different locations. This naming issue existed before this PR, but now the created directory names are also the same.